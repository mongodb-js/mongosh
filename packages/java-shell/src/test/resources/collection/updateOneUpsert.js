// before
db.coll.remove({});
// command
db.coll.updateOne({a: 1}, {$set: {b: 1}}, {upsert: true});
// command
db.coll.find();
// clear
db.coll.drop();
