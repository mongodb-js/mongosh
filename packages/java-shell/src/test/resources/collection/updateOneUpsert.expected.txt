AcknowledgedUpdateResult{matchedCount=0, modifiedCount=0, upsertedId=BsonObjectId{value=<ObjectID>}}
[ { "_id": <ObjectID>, "a": 1, "b": 1 } ]